### DISCUSSION TITLE
Simple idea for volume server System Stats

### DISCUSSION THREAD

**Discussion by thephox1982** [seaweedfs/seaweedfs#6203] [Ideas]
I noticed that the addresses in Masters of the System Stats portion of the volume server web page aren't as links, it would be very handy to not have to select and copy these!

**Comment by chrislusf** [comment]
Send a PR?

**Reply by thephox1982** [reply]
Oh I'm not a contributor, I was just offering a suggestion for anyone who might want to handle it.
